docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -977,6 +977,33 @@ Instructs `saucectl` to run all classes for the suite _except_ those specified h
 
 ---
 
+### `shard`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+When shard is configured as concurrency, saucectl automatically splits the tests by concurrency so that they can run in parallel.
```

Comment:
Taken over from https://docs.saucelabs.com/web-apps/automated-testing/cypress/yaml/v1/#shard for consistency in writing.

```suggestion
When shard is configured as `concurrency`, saucectl automatically splits the tests into several groups (the number of groups is determined by the concurrency setting). Each group will then run as an individual job.
```